usr/share/git/ReleaseNotes.css

```diff
@@ -0,0 +1,373 @@
+html {
```

Comment:
This looks like a copy, but you probably wanted to move the file instead, correct?